src/labkey/security/User.ts

```diff
@@ -68,7 +68,12 @@ export interface CreateNewUserOptions extends RequestCallbackOptions<CreateNewUs
 }
 
 /**
- * Creates a new user account.
+ * Creates a new user account. To create more than one account, provide a semicolon separated list of email addresses.
```

Comment:
The `CreateNewUserResponse` interface should be updated to include `htmlErrors`. Also, please double check that the comments about `message` are still correct.